src/cijoe/core/resources.py

```diff
@@ -45,21 +45,25 @@
 import sys
 from importlib.machinery import SourceFileLoader
 from pathlib import Path
+from typing import Any, Dict, List, Union
 
 try:
     from importlib.resources import files as importlib_files
 except ImportError:
-    from importlib_resources import files as importlib_files
+    from importlib_resources import files as importlib_files_fallback
+
+    importlib_files = importlib_files_fallback
```

Comment:
Is this approach really better? Both are confusing and hard to read, but at least the other is simpler

Reply:
It is what mypy prefers.